packages/admin-panel/src/layout/navigation/UserProfileInfo.jsx

```diff
@@ -25,21 +26,38 @@ const UserEmail = styled(Typography)`
   margin-block-end: 0.9rem;
 `;
 
-export const UserProfileInfo = ({ user, userLinks }) => {
+export const UserProfileInfo = ({ user, profileLink, isFullWidth }) => {
   if (!user) return null;
 
   const name = user.firstName || user.lastName ? `${user.firstName} ${user.lastName}` : null;
+
+  // If the navbar is collapsed and profileLink is not provided, we don't need to render anything
+  if (!isFullWidth && !profileLink) return null;
+
+  // If the navbar is collapsed and profileLink is provided, we render the user's initials as an avatar with a tooltip and a link to the profile
+  if (!isFullWidth) {
+    const initials = `${user.firstName ? user.firstName[0] : ''}${
+      user.lastName ? user.lastName[0] : ''
+    }`;
```

Comment:
```suggestion
  if (!isFullWidth) {
    const initials = `${user.firstName[0] ?? ''}${user.lastName ?? ''}`
```

I think(?) it’s safe to presume JS devs recognise that `someIterable[0]` safely returns `undefined` if it’s out of index